test/EventStore.Client.Tests/ConnectionStringTests.cs

```diff
@@ -136,22 +139,24 @@ public class ConnectionStringTests {
 			Assert.Equal(13, settings.ConnectivitySettings.MaxDiscoverAttempts);
 			Assert.Equal(37, settings.ConnectivitySettings.DiscoveryInterval.TotalMilliseconds);
 			Assert.Equal(NodePreference.Follower, settings.ConnectivitySettings.NodePreference);
-			Assert.Equal(TimeSpan.FromMilliseconds(10), settings.ConnectivitySettings.KeepAlive);
+			Assert.Equal(TimeSpan.FromMilliseconds(10), settings.ConnectivitySettings.KeepAliveInterval);
+			Assert.Equal(TimeSpan.FromSeconds(2), settings.ConnectivitySettings.KeepAliveTimeout);
 #if !GRPC_CORE
 			Assert.NotNull(settings.CreateHttpMessageHandler);
 #endif
 
 			Assert.Equal(330, settings.OperationOptions.TimeoutAfter.Value.TotalMilliseconds);
 			Assert.False(settings.OperationOptions.ThrowOnAppendFailure);
 
-			settings = EventStoreClientSettings.Create("esdb://hostname:4321/?tls=false");
+			settings = EventStoreClientSettings.Create("esdb://hostname:4321/?tls=false&keepAliveInterval=-1&keepAliveTimeout=-1");
```

Comment:
**SUGGESTION:** @thefringeninja can we also add some tests with the permutation of `keepAliveInterval` being set to `-1` and `keepAliveTimeout` having valid value?
It'd be also good to verify if passing `0` is handled properly and if negative value ends up with an exception.